experiment/build/docker_images.py

```diff
@@ -50,6 +55,8 @@ def get_images_to_build(fuzzers, benchmarks):
     for fuzzer in fuzzers:
         for benchmark in benchmarks:
             for name_templ, obj_templ in templates.items():
+                if 'coverage' in fuzzer and 'runner' in name_templ:
+                    continue
```

Comment:
i still don't like this, because of the reasons mentioned [before](https://github.com/google/fuzzbench/pull/543#discussion_r458354778). in particular because of the first point: when we will build images for an experiment, the user will give a list of fuzzers (and benchmarks) in the experiment config, e.g.: [afl, libfuzzer] (and libxml). i'd like to pass that fuzzer list into `get_images_to_build` and get all images that need to be built for that experiment. that should include the `libxml-coverage` image. but for that we'll need to remember to add 'coverage' to the fuzzer list. by keeping the coverage image types in `image_types.yaml`, we never have to think about these special cases, and we won't need extra logic in various parts of code we're about to write.

Reply:
Okay, I understand this point. So I could add the coverage image or a suggestion -

Basically there is only this one condition of not including coverage image.
So my suggestion is if I add a default false flag to the function, which automatically adds coverage to list of fuzzers if not present when manually querying the images. 

> get all images that need to be built for that experiment. that should include the libxml-coverage image.

That should provide this functionality with just one if condition. And in the generation scripts, I'll add flag as true to ignore this behavior. I suggest this because I think adding a coverage image may need more conditions than necessary for GCB one which isn't in this PR and I feel this provides the functionality without having 2 extra images which are basically common with the fuzzer image template.

Wdyt?

Reply:
>  suggest this because I think adding a coverage image may need more conditions than necessary for GCB 

Why will it need more conditions? My understanding is that the current state of this PR needs more conditions. What i'd like too is to avoiding these conditions in the code here in this PR, and in future PRs. I think explicit image types in the image_types.yaml config file are cleaner and easier to reason about than having various conditions and flags in the code.

Reply:
Okay, I understand, I'll the image then.